pkg/defaultRules/defaultRules.yaml

```diff
@@ -1548,3 +1548,273 @@ rules:
       anyOf:
         - $ref: "#/definitions/metadataNamePattern"
         - $ref: "#/definitions/metadataGenerateNamePattern"
+  - id: 55
```

Comment:
missing unit tests for new rules, I suggest starting with creating the tests, to assist the development process.